lib/queries/get-challenges-by-tag.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
```

Comment:
Ok so we will still need to change the getChallengesByTag function to GetByTag('challenges',...)